### PR TITLE
Integrations page

### PR DESCRIPTION
**Pull request by elliott-davis**
tenor-138136232

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-web/app/actions/index.ts [hunk 1]
```diff
@@ -193,13 +200,13 @@ export const signOut = usersActions.signOut;
 
 export const setLayout = uiActions.setLayout;
 
-export const getCookie    = cookieActions.getCookie;
-export const setCookie    = cookieActions.setCookie;
+export const getCookie = cookieActions.getCookie;
+export const setCookie = cookieActions.setCookie;
```
Comment (mgamini) [inline components/builder-web/app/actions/index.ts]: tenor-119640324

File: components/builder-web/app/actions/packages.ts [hunk 2]
```diff
@@ -164,6 +164,7 @@ export function filterPackagesBy(
 }
 
 export function populateDashboardRecent(data) {
+    debugger;
```
Comment (mgamini) [inline components/builder-web/app/actions/packages.ts]: tenor-48091660

File: components/builder-web/app/actions/index.ts (hunk 1 above)
Comment (elliott-davis) [reply components/builder-web/app/actions/index.ts]: tenor-112075757

I blame the auto-save linter

**Pull request merged**